airflow/providers/amazon/aws/operators/s3_list.py

```diff
@@ -65,7 +67,7 @@ class S3ListOperator(BaseOperator):
             )
     """
 
-    template_fields: Iterable[str] = ('bucket', 'prefix', 'delimiter')
+    template_fields: Iterable[str] = ('bucket', 'prefix', 'delimiter' , 'aws_conn_id')
```

Comment:
See previous comment about standardizing.  None of the other operators here use the `: Iterable[str]` type hinting.

Reply:
Well it was done by some else before so i can't comment on that , but i agree , we should standardize type hinting. May i can work on that as part of separate PR.